elf_reader.go

```diff
@@ -373,7 +373,7 @@ func (ec *elfCode) loadFunctions(section *elfSection) (map[string]asm.Instructio
 	r := bufio.NewReader(section.Open())
 
 	// Decode the section's instruction stream.
-	var insns asm.Instructions
+	insns := make(asm.Instructions, 0, section.Size/asm.InstructionSize)
```

Comment:
Can you add the rationale from the PR text to the commit message please? Is there a benchmark which shows the effect?

Also, this will round down if section is not an even multiple. Probably not an issue though?

Reply:
I will add the rationale yes, I don't have a benchmark for now but will add one